Doc/library/math.rst

```diff
@@ -71,8 +71,8 @@ Number-theoretic and representation functions
    Return *n* factorial as an integer.  Raises :exc:`ValueError` if *n* is not integral or
    is negative.
 
-   .. deprecated:: 3.9
-      Accepting floats with integral values (like ``5.0``) is deprecated.
+   .. versionchanged:: 3.10
+      Floating point number (like ``5.0``) is no longer accepted.
```

Comment:
Maybe add "with integral value"?

Reply:
Sure, done. Would you mind to review the updated PR?